README.md

```diff
@@ -1,6 +1,6 @@
 ![](get.png)
 
-*Languages: English (this file), [Brazilian Portuguese](README.pt-br.md).*
+*Languages: [English](README.md), [Brazilian Portuguese](README.pt-br.md), [Polish](README.pl.md).*
```

Comment:
In my opinion, it makes little sense to put a link to the same readme you are already in. If i am already reading english readme, why do i need a link to this same english readme?

Reply:
I agree with that